desk/app/groups.hoon

```diff
@@ -69,7 +69,11 @@
   ++  on-peek   peek:cor
   ::
   ++  on-leave   on-leave:def
-  ++  on-fail    on-fail:def
+  ++  on-fail
+    |=  [=term =tang]
+    ^-  (quip card _this)
+    :_  this
+    [(log-fail:logs /logs our.bowl (fail-event:logs term tang))]~
```

Comment:
You want to handle poke-acks on this wire in `+on-agent`. No-opping is fine, we'll just submit and forget about it.